src/PrestaShopBundle/Security/Voter/PageVoter.php

```diff
@@ -51,6 +55,19 @@ class PageVoter extends Voter
 
     const LEVEL_READ = 1;
 
+    /**
+     * @var CommandBusInterface
+     */
+    private $queryBus;
+
+    /**
+     * @param CommandBusInterface $queryBus
+     */
+    public function __construct(CommandBusInterface $queryBus)
```

Comment:
@matks is this a BC break?

Reply:
Yes it's a BC break 😢 (ping @mickaelandrieu to check ?)

We need to create a new PageVoter and disable the use of the previous. But the class should stay there (and might actually be used by someone who needs to authenticate users using a custom legacy page).

Reply:
actually if I'll keep the `Employee` class backwards compatible, this one can stay as it was before :slightly_smiling_face: